src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms ease;
+    color: black;
```

Comment:
It's better to use SCSS variables for colors. Consider using a variable for the color 'black', e.g., `$black: black;` and then use it like `color: $black;`.